Reverse proxy walkthroughs (#15)
LaunchCodeEducation/aws-basics: content/ec2-web-console/walkthrough/reverse-proxy-websites/dotnet-asp/_index.md

{{% /notice %}}

```bash
which dotnet
```
![Dotnet Package Installed Validation](pictures/dotnet-installed-check.png?classes=border)

You can see that the Package has been installed and is located in the `/usr/bin/` directory.

You can also check that you have the correct version installed with the following command:

```bash
dotnet --version
```

![Check Version of dotnet](pictures/dotnet--version.png?classes=border)

You can see that the version is `6.0.x`.

### Install Caddy

```bash
sudo apt install -y debian-keyring debian-archive-keyring apt-transport-https
```

```bash
curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/gpg.key' | sudo gpg --dearmor -o /usr/share/keyrings/caddy-stable-archive-keyring.gpg
```

```bash
curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/debian.deb.txt' | sudo tee /etc/apt/sources.list.d/caddy-stable.list
```

```bash
sudo apt update
```

```bash
sudo apt install caddy
```

{{% notice note %}}
Verify that caddy was installed using the following commands:
{{% /notice %}}

```bash
which caddy
```

```bash
caddy version
```

![Caddy Installation Validation](pictures/caddy-install-validation.png?classes=border)

### Configure Web Server

> Now that you have cloned the project `build artifacts` in addition to installing the correct `SDK` and `Web Server` you need to configure your `Caddyfile`.

Reviewer: ```suggestion
Now that you have cloned the project `build artifacts` and installed both the correct `SDK` and `Web Server`, you need to configure your Caddyfile.
```
